docs/source/features/passes/pytorch.md

```diff
@@ -183,6 +183,24 @@ This pass only supports Hugging Face transformers PyTorch models. Please refer t
 }
 ```
 
+## SliceGPT
+`SliceGPT` is post-training sparsification scheme that makes transformer networks smaller by applying orthogonal transformations to each transformer layer that leave the model unchanged, and then slicing off the least-significant rows and columns of the weight matrices. This results in speedups and a reduced memory footprint.
```

Comment:
"leave the model unchanged" could be confusing and misleading to some people's opinion. Just reduces size of the weight metrices.

Reply:
Done!